hnn_core/viz.py

```diff
@@ -209,8 +209,17 @@ def plot_extracellular(times, data, tmin=None, tmax=None, ax=None,
     return ax.get_figure()
 
 
-def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
-                color=None, show=True):
+def plot_dipole(dpl,
+                tmin=None,
+                tmax=None,
+                ax=None,
+                layer='agg',
+                decim=None,
+                color=None,
+                average=True,
+                alpha=0.5,
+                linewidth=1.5,
+                show=True):
```

Comment:
try to refrain from stylistic changes in PRs and keep the diff minimal. If necessary, we should discuss and address in a separate PR